Cargo.toml

```diff
@@ -52,8 +52,7 @@ x25519-dalek = "2.0.0"
 x509-parser = "0.16.0"
 yasna = "0.5.0"
 zeroize = "1.8.1"
-nohash-hasher = "0.2.0"
-static_assertions = "1.1.0"
+libp2p-yamux = { version = "0.13.4", package = "yamux" }
```

Comment:
Why are we renaming `yamux` to `libp2p-yamux`? 'libp2p-yamux` is an existing crate in `libp2p`: https://github.com/libp2p/rust-libp2p/blob/master/muxers/yamux/Cargo.toml